packages/core/src/components/callout/_callout.scss

```diff
@@ -23,14 +25,17 @@ Styleguide callout
 
 .#{$ns}-callout {
   @include running-typography();
-  position: relative;
+  display: flex;
   border-radius: $pt-border-radius;
   background-color: rgba($gray3, 0.15);
   width: 100%;
   padding: $pt-grid-size ($pt-grid-size * 1.2) ($pt-grid-size * 0.9);
 
   // CSS API support
   &[class*="#{$ns}-icon-"] {
+    // no flex layout as icon is absolute positioned within padding-left.
+    display: block;
```

Comment:
this change is not necessary anymore, and it allows callouts with icons _and_ headings to avoid the `.pt-callout-text` in HTML. for instance, you missed a callout on the buttons page.

Reply:
that's right, very cool that we don't need it in markdown. can't wait for MDX tho!!